index.html

```diff
@@ -414,41 +420,60 @@ <h2>
         </p>
         <ul>
           <li>An end-user can <dfn data-lt="manual installation">manually</dfn>
-          trigger the <a>installation process</a> through the user agent's
-          <abbr>UI</abbr>.
-          </li>
-          <li>The <a>installation process</a> can occur through an
-          <dfn>automated install prompt</dfn>: that is, a <abbr>UI</abbr> that
-          the user agent presents to the user when, for instance, there are
-          sufficient <a>installability signals</a> to warrant
-          <a>installation</a> of the web application.
-          </li>
-          <li>The <a>installation process</a> can occur through a
-          <dfn>site-triggered install prompt</dfn>: the site can
-          programmatically request that the user agent present an install
-          prompt to the user. The user agent MAY restrict the availability of
-          this feature to cases where, for instance, there are sufficient
+          trigger the installation process through the user agent's
+            <abbr title="User Interface">UI</abbr>, directly invoking the steps
+            to <a>present an install prompt</a>.
+          </li>
+          <li>The installation process can occur through an <dfn>automated
+          install prompt</dfn>: that is, a <abbr>UI</abbr> that the user agent
```

Comment:
ReSpec will add abbr automagically. 

```suggestion
          install prompt</dfn>: that is, a UI that the user agent
```